### DISCUSSION TITLE
PCL 为什么可以登录正版验证？它的原理是什么？

### DISCUSSION THREAD

**Discussion by ghost** [Meloong-Git/PCL#2395] [讨论]
PCL 为什么可以登录正版验证？它的原理是什么？

**Comment by zkitefly** [comment]
因为可以，就是可以啊（

**Comment by yaxuan97** [comment]
微软有一套专门的API进行账号验证。
参考：
MojangAPI Doc -> URL[https://mojang-api-docs.gapple.pw/authentication/msa](url)

**Comment by hejiehao** [comment]
我觉得PCL2的正版验证应该和Minecraft Wiki上的类似
https://minecraft.fandom.com/zh/wiki/%E6%95%99%E7%A8%8B/%E7%BC%96%E5%86%99%E5%90%AF%E5%8A%A8%E5%99%A8?so=search
